xml/cl.xml

```diff
@@ -7614,5 +7617,15 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_UNORM_INT14X2_EXT"/>
             </require>
         </extension>
+        <extension name="cl_khr_spirv_queries" revision="1.0.0" supported="opencl" ratified="opencl">
+            <require>
+                <type name="CL/cl.h"/>
+            </require>
```

Comment:
Do we really need these lines?

Reply:
I'm honestly not sure.

They're present for every extension, though, so if we decide that we don't need them, I'd prefer to remove them all in a separate PR.